deployments/polygon/usdc/migrations/1675200105_raise_polygon_supply_caps.ts

```diff
@@ -1,8 +1,12 @@
+import { Contract } from 'ethers';
 import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
 import { migration } from '../../../../plugins/deployment_manager/Migration';
 import { calldata, exp, getConfigurationStruct, proposal } from '../../../../src/deploy';
 import { expect } from 'chai';
 
+const ERC20PredicateAddress = '0x40ec5B33f54e0E8A33A975908C5BA1c14e5BbbDf';
+const RootChainManagerAddress = '0xA0c68C638235ee32657e8f720a23ceC1bFc77C77';
```

Comment:
Addresses found here: https://static.matic.network/network/mainnet/v1/index.json

Reply:
Yeah looks like that's the `DummyERC1155` on Polygon. So they probably deployed these two contracts using the same EOA and nonce.